pkg/router/template/router.go

```diff
@@ -512,12 +512,27 @@ func (r *templateRouter) writeConfig() error {
 		r.state[k] = cfg
 	}
 
+	data, err := ioutil.ReadFile("/proc/sys/net/ipv4/neigh/default/gc_thresh3")
+	if err != nil {
+		glog.Warningf("Error reading ARP neighbour information: %s", err)
+	}
+	arpcache, err := strconv.Atoi(string(data[:len(data)-1]))
```

Comment:
Shouldn't we only do that if we successfully read?

We should expect that the arp size may not be readable, log the error, and then allow the router to proceed.  In this case we should not check the number of services.  Perhaps when arpcache is 0?